brut.apktool/apktool-cli/build.gradle

```diff
@@ -58,6 +58,7 @@ task proguard(type: ProGuardTask, dependsOn: shadowJar) {
         libraryjars "${System.properties['java.home']}/jmods/java.xml.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
         libraryjars "${System.properties['java.home']}/jmods/java.desktop.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
         libraryjars "${System.properties['java.home']}/jmods/java.sql.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
+        libraryjars "${System.properties['java.home']}/jmods/java.scripting.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
```

Comment:
delete 58,59,60,61
add 58
`        libraryjars "${System.properties['java.home']}/jmods"`

Reply:
Are you sure you want to do two changes in one PR? (deprecation resolution and changing from a limited set of jmod references to all and any). I can see a potential issue with folk who may have locally customised their jmod definitions getting a different set of results to folk with an unmodified Java install.